### ISSUE TITLE
new error in CI build process which I don't understand

### ISSUE THREAD

**Issue by thareUSGS** [pds-data-dictionaries/PDS4-LDD-Issue-Repo#136]
This was a simple update to IMG, but building the LDD now errors. It looks like something else was updated in a different script for LDD building. Perhaps related to a log directory? I'm not sure how to track that down or if I just need to add a log directory somewhere.

https://github.com/pds-data-dictionaries/ldd-img/runs/1432294382

**Comment by jordanpadams** [comment]
@thareUSGS FYI I am looking into this

**Comment by jordanpadams** [comment]
@thareUSGS so I think this was just an issue with your branch being slightly out of sync with master. per the last merge for the IM release, I made a couple updates to the CI/CD that were not on your `v1820` branch. I rebased the branch and created a new one for your here: https://github.com/pds-data-dictionaries/ldd-img/tree/v1820_rebased . So I would just work off of that.

the ["long-running branch" workflow](https://git-scm.com/book/en/v2/Git-Branching-Branching-Workflows) you are using can work great, just make sure you [rebase](https://git-scm.com/book/en/v2/Git-Branching-Rebasing) on occasion to make sure you pick up any changes that happen on master.
